### PR TITLE
add quickstart instructions to readme

### PR DESCRIPTION
**Pull request by jnnnnn**
I had to google for how to set things up, I figured it'd be helpful to have some quickstart instructions in the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,9 +47,19 @@ Compatibility. We use more than one operating system and you probably do, too. W
 
 Communication. Everything we do is in the open. Our issue tracker will let you see if others are having the same problem you're having and will allow you to add additional information. You will also be able to see when progress is made and how the issue gets resolved.
 
+### Usage
+
+Install and run barrier on each machine that will be sharing. On the machine with the keyboard and mouse, make it the server.
```
Comment (xkr47) [inline README.md:52]:
```suggestion
Install and run barrier on each machine that will be sharing. Run the server on the machine whose keyboard and mouse you want to share and the client on all other machines.
```

**Review by xkr47 — COMMENTED** [review]
Great updates, I was spending much time setting up the client and server the wrong way around :)

**Comment by dayne** [comment]
This is great. @jnnnnn you should add in @xkr47 suggestions and hopefully one of the admins merges as this is a small step towards the [improving documentation goal](https://github.com/debauchee/barrier/projects/2).

**Comment by p12tic** [comment]
I removed some whitespace changes, rewrapped lines and rebased on top of latest master.

This PR looks good to me, thanks!

**Review by p12tic — APPROVED** [review]

**Pull request merged**